designer/server/src/test/scala/pl/touk/nussknacker/ui/integration/DictsFlowTest.scala

```diff
@@ -32,9 +32,43 @@ class DictsFlowTest
   private val EndNodeId      = "end"
   private val Key            = "foo"
   private val Label          = "Foo"
+  private val DictId         = "dict"
 
   override def nuTestConfig: Config = ConfigWithScalaVersion.TestsConfigWithEmbeddedEngine
 
+  test("query dict entries by label pattern") {
```

Comment:
The purpose of these *Flow* tests is to show e2e a flow that user follows. Thanks to that we avoid the situation that something works locally, but after integrating all mechanisms, it makes no sense. Do you know this meme?
![image](https://github.com/TouK/nussknacker/assets/534341/592b7cee-ebe3-4a9a-86a0-3f179eb0dbed)

We need a test that would follow the flow:
- user creates a scenario
- user uses the editor to query for a key
- user saves the scenario with the provided key
- user invokes a test based on this scenario

Current, tests do similar thing using dictionary variables (take a look at usage of `saveProcessAndTestIt`). They had a gap in the second step, because when we wrote them, the suggestion mechanism was on the FE side. When we moved it into BE side, we forgot to add this missing part.

I'm not sure it this test will work correctly in the 4th step - we don't want to Flink validate dictionaries.

Reply:
created such a test